Cargo.toml

```
@@ -30,7 +30,7 @@ bincode = "1.3"
 bitvec = "1.0"
 chrono = "0.4"
 clap = { version = "3.2", optional = true }
-crc = "=2.0.0"
+crc = "3.0"
```

It is better to require concrete version of crate, because it theoretically can change calculation logic. Every update should be validated explicitly
```suggestion
crc = "=3.0.0"
```